MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -7077,5 +7087,80 @@ private void ul2as()
         ///     Signature: int errcod;
         /// </summary>
         private ReadOnlySpan<byte> errcod => Module.Memory.GetVariablePointer("ERRCOD").Data;
+
+        /// <summary>
+        ///     Determines the profanity level of a string
+        ///
+        ///     MBBSEmu doesn't support multiple profanity levels, so the default value of 0 is returned.
+        /// 
+        ///     Signature:  int profan(char *string);
+        /// </summary>
+        private void profan()
+        {
+            Registers.AX = 0;
+        }
+
+        /// <summary>
+        ///     Expect a YES or NO from the user
+        ///
+        ///     Signature: int yesno=cncyesno();
+        /// </summary>
+        private void cncyesno()
+        {
+            //Get Input
+            var inputLength = Module.Memory.GetWord("INPLEN");
+            var inputPointer = Module.Memory.GetVariablePointer("INPUT");
+            var inputString = Module.Memory.GetArray(inputPointer, inputLength);
+            var inputStringComponents = Encoding.ASCII.GetString(inputString).ToUpper().Split('\0');
+
+            if (inputStringComponents.Length == 0)
+            {
+                Registers.AX = 0;
+                return;
+            }
+
+            switch (inputStringComponents[0])
+            {
+                case "YES":
+                case "Y":
+                    Registers.AX = 'Y';
+                    break;
+                case "NO":
+                case "N":
+                    Registers.AX = 'N';
+                    break;
+                default:
+                    Registers.AX = 0;
```

Comment:
I thought this only returns 'Y' or 'N', at least the docs seem to indicate.

Reply:
After running a test module, determined `cncyesno()` returns the 1st character of the command, regardless if it's YES or NO. Updated code to reflect this.